### PR TITLE
Moved all strings to a Resource file

### PR DESCRIPTION
**Pull request by Char0394**
### Description of Change ###

Moved all the strings of the sample to a Resource file. 
https://github.com/xamarin/XamarinCommunityToolkit/issues/165

*Now we have all the strings in a centralized place is easier to add multilingual support, I think we can open different issues per language, to have all the community involved.* 

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [X] Rebased on top of main at time of PR
- [X] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

**Comment by almirvuk** [comment]
This is great!

File: XamarinCommunityToolkitSample/Pages/Converters/ItemSelectedEventArgsPage.xaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
              xmlns:behaviors="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Behaviors;assembly=Microsoft.Toolkit.Xamarin.Forms"
              xmlns:converters="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Converters;assembly=Microsoft.Toolkit.Xamarin.Forms"
              x:Class="Microsoft.Toolkit.Xamarin.Forms.Sample.Pages.Converters.ItemSelectedEventArgsPage"
-             xmlns:pages="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Pages">
+             xmlns:pages="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Pages" xmlns:resources="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Resx">
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample/Pages/Converters/ItemSelectedEventArgsPage.xaml:8]: please move xmlns:resources to the new line

File: XamarinCommunityToolkitSample/Pages/Converters/ItemTappedEventArgsPage.xaml [hunk 2]
```diff
@@ -5,7 +5,7 @@
                 xmlns:behaviors="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Behaviors;assembly=Microsoft.Toolkit.Xamarin.Forms"
                 xmlns:converters="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Converters;assembly=Microsoft.Toolkit.Xamarin.Forms"
                 x:Class="Microsoft.Toolkit.Xamarin.Forms.Sample.Pages.Converters.ItemTappedEventArgsPage"
-                xmlns:pages="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Pages">
+                xmlns:pages="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Pages" xmlns:resources="clr-namespace:Microsoft.Toolkit.Xamarin.Forms.Sample.Resx">
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample/Pages/Converters/ItemTappedEventArgsPage.xaml:8]: Same here

**Comment by AndreiMisiukevich** [comment]
Overall - Well done!

**Comment by sthewissen** [comment]
Awesome!

**Pull request merged**